### PR TITLE
Fixed the errored response of python-flask server

### LINKED ISSUE #270: To fix the API response going to the SUSI Android App
**Describe the bug**
Now the APP is connected and is sending instructions to SUSI smart speaker, but is not getting any successful response and hence after successful installations, error message is still being shown in the app. (Screenshots added below)

**Expected behavior**
To fix the bugs

**Screenshots**
2018-07-17-032755_1024x768_scrot

37219759_1980224808676445_8538568733054992384_n

**Desktop (please complete the following information):**
Raspbian

**Smartphone (please complete the following information):**
Android v8.0.0 

**Additional context**
I would like to take up the issue

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #270 

Now, the response code is being sent from the server and the reply is also in JSON. It fixes the error caused in the app

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@prateekiiest @batbrain7 , please have a look too

File: access_point/server/server.py [hunk 1]
```diff
@@ -17,20 +18,30 @@ def install():
...
+    return resp # pylint-enable
 
 @app.route('/wifi_credentials/<wifissid>/<wifipassd>')
 def wifi_config(wifissid,wifipassd):
     wifi_ssid = wifissid
     wifi_password = wifipassd
-    subprocess.call(['sudo', 'bash', access_point_folder + '/wifi_search.sh', wifi_ssid, wifi_password])  #nosec #pylint-disable type: ignore
-    return 'Wifi Configured' # pylint-enable
+    subprocess.call(['sudo', 'bash', wifi_search_folder + '/wifi_search.sh', wifi_ssid, wifi_password])  #nosec #pylint-disable type: ignore
+    display_message = {"Wifi":"Configured"}
```
Comment (prateekiiest) [inline access_point/server/server.py]: Looks good to me.
Have you tested it locally?

Comment (stealthanthrax) [reply access_point/server/server.py]: Yes @prateekiiest  , its working now

**Comment by stealthanthrax** [comment]
@cweitat @woshikie , please have a look

**Pull request merged**